aci/utils.go

```diff
@@ -34,6 +34,23 @@ func toStringList(configured []interface{}) []string {
 	return vs
 }
 
+func setRelationAttribute(d *schema.ResourceData, relation string, data interface{}) {
+	if _, ok := d.GetOk("annotation"); ok {
```

Comment:
I am not sure why we do this condition here. What is it trying to filter?

Reply:
Basically we need a filter to know whether this function is called be Terraform or Terraformer import.

- If it is called during terraform execution, annotation (common attribute in all resources) will be always be set as it has a default value too. If we skip this filter, Terraform will show changes for all the relationships having default values set by server.
-  If it is called during terraformer import, no attributes will be set. So it will simply read the values from the server and set it.

Reply:
How would this behave if a user set the annotation to an empty string in his plan?
There is also classes that do not support annotation. Today we keep it in the schema but not put it in the payload does that also work with those?

Reply:
> How would this behave if a user set the annotation to an empty string in his plan?

if value is set to empty terraform will use the last applied value. It will not apply empty value.

> There is also classes that do not support annotation. Today we keep it in the schema but not put it in the payload does that also work with those?

Yes it will work. It only checks if annotation is set in tf-state. As far as it has default value in the schema, there should not be any issues.

Reply:
Sure

Reply:
added comments, let me know if it needs to be updated.